Sources/TensorFlow/Operators/Math.swift

```diff
@@ -1610,3 +1612,433 @@ internal extension Tensor where Scalar: TensorFlowFloatingPoint {
         return _vjpMatmul(lhs, rhs)
     }
 }
+
+
+//===------------------------------------------------------------------------------------------===//
```

Comment:
Before the actual code reviews do start, maybe we should just make sure the style guidelines are out of the way. Could you just make sure that this and the one below `einsum and helper functions`  fits into 100 columns?